Dish.cs

```diff
@@ -0,0 +1,38 @@
+namespace CourseApp
+{
+    using System;
+
+    public class Dish
+    {
+        private int price;
+
+        public string Name { get; set; }
+
+        public int Weight { get; set; }
+
+        public int Price
+        {
+            get
+            {
+                return this.price;
+            }
+
+            set
+            {
+                if (value > 0 & value < 10000)
+                {
+                    price = value;
+                }
+            }
+        }
+
+        public object ProgramDish(string name, int weight, int price)
+        {
+            Console.WriteLine("dish constructor called");

```

Comment:
i think -that it's not the buiseness of this method to write something to the console